doc/Command-Reference.md

```diff
@@ -1970,7 +2078,7 @@ This sub-section explains the various IPv6 protocol specific show commands that
 This command displays either all the IPv6 route entries from the routing table or a specific IPv6 route.
 
   - Usage:  
-    show ipv6 route [<ipv6-address>]
+    show ipv6 route <ipv6_address>

```

Comment:
Why remove the square brackets? Is <ipv6_address> no longer optional?

Reply:
It was removed by mistake. There was an issue in using '<' within square brackets '[' . While solving it, this one got missed. It is now addressed properly in all places by using a backslash '\' before the '<'.